Implement default key

allevo opened the pull request:
Implement https://github.com/fastify/fast-json-stringify/issues/87

The jsonschema can define `default` is the property is not given. This PR implements that feature.

delvedor commented on test/defaults.test.js:
```diff
@@ -0,0 +1,342 @@
+    const output = stringify(toStringify)
+    const outputUglify = stringifyUgly(toStringify)
+
+    t.deepEqual(output, JSON.stringify(expected))
```
I would use `strictEqual` here since we are working with strings.

allevo commented:
@delvedor 

```
$ git checkout master 
Switched to branch 'master'
$ npm run benchmark
> node bench.js

JSON.stringify array x 2,425 ops/sec ±3.44% (85 runs sampled)
fast-json-stringify array x 2,941 ops/sec ±1.21% (87 runs sampled)
fast-json-stringify-uglified array x 2,911 ops/sec ±1.25% (89 runs sampled)
JSON.stringify long string x 9,423 ops/sec ±2.61% (83 runs sampled)
fast-json-stringify long string x 9,810 ops/sec ±0.95% (89 runs sampled)
fast-json-stringify-uglified long string x 9,533 ops/sec ±1.75% (89 runs sampled)
JSON.stringify short string x 3,797,522 ops/sec ±2.32% (90 runs sampled)
fast-json-stringify short string x 7,643,835 ops/sec ±0.83% (92 runs sampled)
fast-json-stringify-uglified short string x 7,599,888 ops/sec ±1.28% (91 runs sampled)
JSON.stringify obj x 1,334,082 ops/sec ±1.04% (89 runs sampled)
fast-json-stringify obj x 2,788,622 ops/sec ±1.38% (89 runs sampled)
fast-json-stringify-uglified obj x 2,752,812 ops/sec ±1.38% (88 runs sampled)
````

```
$ git checkout feature/implement-defaults 
Switched to branch 'feature/implement-defaults'
$ npm run benchmark
> node bench.js

JSON.stringify array x 2,523 ops/sec ±1.52% (88 runs sampled)
fast-json-stringify array x 2,953 ops/sec ±2.30% (87 runs sampled)
fast-json-stringify-uglified array x 2,916 ops/sec ±3.07% (87 runs sampled)
JSON.stringify long string x 9,673 ops/sec ±1.48% (91 runs sampled)
fast-json-stringify long string x 9,744 ops/sec ±1.11% (90 runs sampled)
fast-json-stringify-uglified long string x 9,750 ops/sec ±1.23% (89 runs sampled)
JSON.stringify short string x 3,742,537 ops/sec ±2.05% (88 runs sampled)
fast-json-stringify short string x 7,290,348 ops/sec ±5.00% (86 runs sampled)
fast-json-stringify-uglified short string x 7,645,764 ops/sec ±0.85% (91 runs sampled)
JSON.stringify obj x 1,326,023 ops/sec ±1.27% (88 runs sampled)
fast-json-stringify obj x 2,769,480 ops/sec ±1.59% (88 runs sampled)
fast-json-stringify-uglified obj x 2,790,791 ops/sec ±1.75% (89 runs sampled)
```

No changes

delvedor commented:
I've merged too quickly :D
Can you open another pr to update the docs?
